### ISSUE TITLE
Ferocious Inspiration [hunter talent]

### ISSUE THREAD

**Issue by amaroc** [OregonCore/OregonCore#1545]
**Revision**:
OregonCore Rev: 3602

**Operating system**:
ubnt 16.04

**Current behavior**: 
It`s give buff, but only once for game session untill you relog.  So it`s broke on 99.99%

**Expected behavior**: 
Give buff for all party (scalable for tallent points) after  pet scores a critical hit.

**Steps to reproduce the problem**:
Get pet, go in to battle with this talents.

**Comment by PadreWoW** [comment]
very old rev
try the last one

**Comment by amaroc** [comment]
I have a few changes important to me relating to PVE and some others little "fix". Wich command (git) i can use to save them and get updated version in other parts of oregon? Howis it called revision?

**Comment by amaroc** [comment]
@PadreWoW i dont see a big changes after this revision (i get 3602 in december 2016). Very old its maybe trinity where updates every day :)

**Comment by PadreWoW** [comment]
Last OC commit was 3 days ago:

 Update appveyor.yml and appveyor's build status image
Zaffy committed 3 days ago
https://github.com/OregonCore/OregonCore/commit/a490255aa51b813017a0957a19523dece34ab0e9

**Comment by amaroc** [comment]
Oh... maybe it`s same like update README.MD? ) I mean i dont see BIIIG changes wich touch problem that i see on 3602.

**Comment by PadreWoW** [comment]
damned......you don't understand ANYTHING that i'm saying
i sad that there are MANY MANY new commits after 3602!
just look:
https://github.com/OregonCore/OregonCore/commits/master

There are 3797 commits - that means there are 195 commits after yours!

**Comment by amaroc** [comment]
Sorry, Russian(offtop): ну да пропустил март не мониторил сильно, но не уверен что там все о чем я писал "пофикшено" бегло взглянул про хантера, лока итп нет. Максимум про агро на трупе. @PadreWoW какая команда не подскажешь у гита что бы провести "ревизию" и сохранить мои изменения и получить апдейты?

**Comment by PadreWoW** [comment]
хз я качаю руками и вношу патчи тоже руками
гитом я пользуюсь только для тестирования и отката патчей на чистом ядре для обнаружения проблемных

**Comment by Selphius** [comment]
duplicate of issue: https://github.com/OregonCore/OregonCore/issues/819
